cfn-resources/encryption-at-rest/mongodb-atlas-encryptionatrest.json

```diff
@@ -60,31 +50,37 @@
         "AwsKms",
         "ProjectId"
     ],
```

Comment:
We can add requriedParameters as well. Not mandatory but good to have validations before hitting the handlers.

Reply:
required is at line 59-62 of this file and contains `AwsKms` and `ProjectId`. Should I add something else?